betterconf/config.py

```diff
@@ -14,7 +14,7 @@ class BetterconfError(Exception):
 class VariableNotFoundError(BetterconfError):
     def __init__(self, variable_name: str):
         self.var_name = variable_name
-        self.message = "Variable ({}) is not found".format(variable_name)
+        self.message = "Variable ({}) was not found in the environment".format(variable_name)
```

Comment:
user could have another from environment provider, so this addition (maybe except changing the tense) is kinda useless